R/mod_mitigators_summary_server.R

```diff
@@ -6,9 +6,11 @@ mod_mitigators_summary_server <- function(id, age_sex_data, params) {
     mitigators_summary <- shiny::reactive({
       year <- as.character(shiny::req(params$start_year))
 
-      strategy_codes <- app_sys("app", "data", "mitigator-codes.Rds") |>
-        readr::read_rds() |>
-        dplyr::select("strategy", "strategy_name", "mitigator_code")
+      strategy_codes <- readr::read_csv(
+        app_sys("app", "data", "mitigator-codes.csv"),
+        col_select = c("strategy", "strategy_name", "mitigator_code"),
+        show_col_types = FALSE
```

Comment:
Why not col_types="..."? Would make sense to be specific on the types here

Reply:
All cols are char. Was preventing {readr} printing to the console. Replaced with `col_types` call instead. 👍